test/dataframes.jl

```diff
@@ -4,64 +4,191 @@ module TestDataFrames
 using Test
 using DataFrames
 using DataFramesMeta
+using Statistics
+
+const ≅ = isequal
+
+df = DataFrame(
```

Comment:
it is better to wrap all variables inside a `@testset` (even if some definitions are repeated) as it makes `@testset` easier to maintain in the long run.

Reply:
I've implmented that where I can but in many cases the use of `@eval` means i need the data in global scope.

Reply:
ah - right!